src/Features.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * Provide feature detection
+ * @class Features
+ */
+export class Features {
+  /**
+   * If the browser has flash
+   */
+  static get flash() {
```

Comment:
Do we actually care anymore?

Reply:
i know i dont and neither should anyone else 😄